src/sentry/api/endpoints/organization_auth_providers.py

```diff
@@ -22,14 +21,11 @@ def get(self, request, organization):
         """
         provider_list = []
         for k, v in manager:
-            is_saml = issubclass(v, SAML2Provider)
-            if is_saml and not HAS_SAML2:
-                continue
-
-            feature = v.required_feature
-            if feature and not features.has(feature, organization, actor=request.user):
-                continue
-
-            provider_list.append((k, v.name, is_saml))
+            provider_list.append({
```

Comment:
Is this endpoint documented/public? Did we check to see if any other views are using it?

Reply:
This API isn't documented no, and this is the only view using it.